source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: post
+title: "Statie 4: How to Create The Simplest Blog"
+perex: '''
+Statie is very powerful tool for creating small sites. But you will use just small part of it's features, having just micro-sites. How to get to full 100%? <strong>Build a blog</strong>.
+    <br><br>
+    Today I will show you, <strong>how to put your first post</strong>.
+'''
+lang: en
+---
+
+
+## Create a Blog Page
+
+This might be the simplest page to show all you posts:
+
+
+```html
+<!-- /source/blog.latte -->
+
+---
+layout: default
+---
+
+❴block content❵
+    <h1>Shouts Too Loud from My Hearth</h1>
+
+    ❴foreach $posts as $post❵
+        <a href="/❴$post['relativeUrl']❵/">
+            <h2>❴$post['title']❵</h2>
+        </a>
+    ❴/foreach❵
+❴/block❵
+```
+
+### You already see
+
+- that all posts are in stored in `$posts` variable
```

Comment:
are in stored in ? nevím jak to myslíš, třeba to máš dobře, ale radši se ptám :-)